sssom/writers.py

```diff
@@ -84,6 +84,13 @@ def write_rdf(
     print(t.decode(), file=file)
 
 
+# todo: not sure the need for serialization param here; seems superfluous for some of these funcs
+def write_fhir_json(msdf: MappingSetDataFrame, output: TextIO, serialisation) -> None:
```

Comment:
will you add the reader as well?

Reply:
Will not be adding to this PR, as I have to get these tests to pass / add a test, and we need to merge this while refactoring, but I have it in the works for a future PR.